tt_dbutil.py

```diff
@@ -202,4 +197,22 @@ def create_connection(db_file) -> sqlite3.Connection:
         print("sqlite ERROR in create_connection() -- ", sqlite_err)
     return connection
 
+def db_commit(db:sqlite3.Connection):
+    """Just a database commit.  Only here for completeness."""
+    db.commit()
+
+def db_update(db:sqlite3.Connection,update_sql:str,commit:bool=True) -> bool:
+    """Execute a SQL UPDATE statement.  T/F indicates success.
+
+    (This could be any SQL statement, but the error checking and
+    return is based on assumption it's an UPDATE)
+    """
+    try:
+        db.cursor().execute(update_sql)
+        if commit:
+            db.commit()
```

Comment:
Why not use your `db_commit()` here rather than `db.commit()`?

Reply:
db_commit was an initial idea - moved it to the dbutil module for completeness - for one thing it removes a dependency in cgi-reports to sqlite3.  I will remove db_commit in next rev?